src/main/kotlin/org/rust/lang/core/psi/ext/RsTypeParameter.kt

```diff
@@ -24,6 +24,13 @@ val RsTypeParameter.bounds: List<RsPolybound> get() {
     return typeParamBounds?.polyboundList.orEmpty() + whereBounds
 }
 
+val RsTypeParameter.isSized: Boolean get() {
+    return stub?.isSized ?: bounds.none {
+        if (it.q == null) return@none false
+        it.bound.traitRef?.resolveToTrait?.isSizedTrait ?: false
```

Comment:
For dealing with nullable booleans, ` == true` is more idiomatic: https://github.com/JetBrains/kotlin-web-site/blob/465783bf395acc5d34f089291eca594ef3d7d4ce/pages/docs/reference/coding-conventions.md#using-nullable-boolean-values-in-conditions